src/api/objects/ObjectAPI.js

```diff
@@ -200,8 +201,14 @@ define([
             result = Promise.resolve(true);
         } else {
             if (domainObject.persisted === undefined) {
-                this.mutate(domainObject, 'persisted', domainObject.modified);
-                result = provider.create(domainObject);
+                domainObject.persisted = domainObject.modified;
+                result = new Promise((resolve) => {
+                    savedResolve = resolve;
+                });
+                provider.create(domainObject).then((response) => {
+                    this.mutate(domainObject, 'persisted', domainObject.modified);
```

Comment:
This is a good hack, but it results in 2 requests to the backend, which is ok for now.